### ISSUE TITLE
Documentation issue for sha1.(

### ISSUE THREAD

**Issue by billdenney** [eddelbuettel/digest#132]
I didn't make this clear enough when I wrote the sha1.( method.  There is a documentation issue which I could not figure out on how to add the `sha1.(` method to the .Rd file.  When checking the code, I get the following error:

```
> checking Rd \usage sections ... WARNING
  Bad \usage lines found in documentation object 'sha1':
    <unescaped bksl>method{sha1}{"("}(x, digits = 14, zapsmall = 7, ..., algo = "sha1")
  
  Functions with \usage entries need to have the appropriate \alias
  entries, and all their arguments documented.
  The \usage entries must correspond to syntactically valid R code.
  See chapter 'Writing R documentation files' in the 'Writing R
  Extensions' manual.
```

It relates to this line of documentation:

https://github.com/eddelbuettel/digest/blob/f0b6fa91d0f2efe4223a677ad41018885b92a248/man/sha1.rd#L51

I tried several variants of escaping the parenthesis (`"("`, `\(`, and `\\(`) without success.

**Comment by eddelbuettel** [comment]
Maybe ask on r-package-devel to get in front of Duncan Murdoch and others?

**Comment by billdenney** [comment]
Duncan replied to use back-ticks: ``\method{sha1}{`(`}(x, digits = 14, zapsmall = 7, ..., algo = "sha1")``.

To be closed when #131 is merged.
